acceptance/README.md

```diff
@@ -1,257 +1,460 @@
-
-### Cleaning Up Preserved Hosts
-
-If you run a number of jobs with --preserve_hosts or vi ci:test_and_preserve_hosts, you may eventually generate a large number of stale vms.  They should be reaped automatically by qa infrastructure within a day or so, but you may also run:
-
-    bundle exec rake ci:release_hosts
-
-to clean them up sooner and free resources.
-
-There also may be scenarios where you want to specify the host(s) to release. E.g. you may want to release a subset of the hosts you've created. Or, if a test run terminates early, ci:release_hosts may not be able to derive the name of the vm to delete. In such cases you can specify host(s) to be deleted using the HOST_NAMES environment variable. E.g.
+Always clean up after yourself
+```
+bundle exec beaker destroy
+```
 
-    HOST_NAMES=lvwwr9tdplg351u bundle exec rake ci:release_hosts
-    HOST_NAMES=lvwwr9tdplg351u,ylrqjh5l6xvym4t bundle exec rake ci:release_hosts
+-------------
 
-### Syncing local development code with VMPooler Windows machines
+## Configuration
+### Environment Variables
+A detailed description of the available environment variables can be found by
+running `bundle exec rake ci:help`. This will print a list of both required and
+optional environment variable with short descriptions on how they are used.
+Please review all of these options as they will impact how your test servers
+are provisioned. This rake task is the most up to date source for this
+information.
+
+### Customizing Test Targets
+If you are using the vmpooler hypervisor and the internal pooling api, you can
+customize the platforms to test on using the HOSTS environment variable. The
+internal pooling api is only available to Puppet, Inc. employees. You'll set the
+`HOSTS` environment variable to the host string you want to test, such as
+`HOSTS=redhat7-64ma-windows2012r2-64a`.
+
+If you are unsure of the syntax you need, you can verify the host string against
+the options in
+[beaker hostgenerator](https://github.com/puppetlabs/beaker-hostgenerator/blob/master/lib/beaker-hostgenerator/data.rb). Be sure to use the key from the hash structure.
+You can safely ignore the data associated with the key.
+Generally, this string will be in the format
+`{platform}{version}-{architecture}{role/s}`. You will most often use either the
+agent (a) or master (m) role, but you can find a list of available roles in
+[beaker hostgenerator](https://github.com/puppetlabs/beaker-hostgenerator/blob/master/lib/beaker-hostgenerator/roles.rb).
+Multiple hosts in the string are separated with a dash(`-`). You must have at
+least one agent and at least one master.
+
+Be careful not to confuse the different host string formats. We have different
+tools that expect the host string to be in different forms. For example,
+`packaging_platform` is specific to how [Vanagon](https://github.com/puppetlabs/vanagon)
+parses that string.
+
+### hosts.yaml
```

Comment:
This section needs more clarity
- beaker requires hosts.yaml to exist in order to run
- the automation will create this file, and clean it up, as a part of the normal beaker run. It defaults to the vmpooler hypervisor
- if you need to customize this file, you must use beaker-hostgenerator to generate a hosts and write it to a local file
- if you have a local hosts.yaml file that you have either generated or edited, you must tell beaker where to find that file by setting `HOSTS=path/to/hosts.yaml`

Reply:
Clarify which situations under which you would want to alter or modify the hosts file. These scenarios where the default does not work for a user.